### PR TITLE
Add Gajim plugin: url image preview

### PR DESCRIPTION
**Pull request by a-andre**

### CODE REVIEW

**Comment by nedrichards** [comment]
bot, build org.gajim.Gajim.Plugin.url_image_preview

File: org.gajim.Gajim.Plugin.url_image_preview.metainfo.xml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="addon">
+  <id>org.gajim.Gajim.Plugin.url_image_preview</id>
+  <extends>org.gajim.Gajim.desktop</extends>
+  <name>URL Image Preview Plugin</name>
```
Comment (barthalion) [inline org.gajim.Gajim.Plugin.url_image_preview.metainfo.xml]: It should probably contain project name here.

Comment (a-andre) [reply org.gajim.Gajim.Plugin.url_image_preview.metainfo.xml]: I don't see why it should contain the project name. Plugins are only listed as part of the project page (at least in gnome-software).

The other plugins available via flathub (https://github.com/flathub/de.wolfvollprecht.UberWriter.Plugin.TexLive and https://github.com/flathub/io.github.Hexchat.Plugin.Perl) also don't contain the project name there.

Comment (barthalion) [reply org.gajim.Gajim.Plugin.url_image_preview.metainfo.xml]: Ah, fair point. Thought it also shows up in search results.

**Comment by barthalion** [comment]
Repository has been created: https://github.com/flathub/org.gajim.Gajim.Plugin.url_image_preview

Thanks!

**Comment by barthalion** [comment]
Checksum is wrong so it fails to build.

**Pull request closed without merging**